src/importexport/midi/internal/midiimport/importmidi_tempo.cpp

```diff
@@ -102,7 +107,9 @@ void setTempoToScore(Score* score, int tick, double beatsPerSecond)
 
 double roundToBpm(double beatsPerSecond)
 {
-    return qRound(beatsPerSecond * 60.0) / 60.0;
+    auto conf = mu::modularity::ioc()->resolve<mu::iex::midi::IMidiImportExportConfiguration>("iex_midi");
+    bool round = conf ? conf->roundTempo() : true;
+    return round ? qRound(beatsPerSecond * 60.0) / 60.0 : (beatsPerSecond * 60.0) / 60.0;
```

Comment:
You can simplify this calculation a little bit:
```suggestion
    return round ? qRound(beatsPerSecond * 60.0) / 60.0 : beatsPerSecond;
```

The `* 60.0 / 60.0` is only relevant when rounding: we need to convert from beats per second to beats per minute, because we want to round to whole beats per minute, not necessarily whole beats per second. That's why we do that. But when not rounding, it's meaningless extra work.